test.py

```diff
@@ -1,22 +1,20 @@
-import argparse
-
+import typer
 from ultralytics import YOLO
 
 
-def predict_yolo(path_to_weights, path_to_test):
-    model = YOLO(path_to_weights)
-    model.predict(source=path_to_test)
-
-
-def main():
-    parser = argparse.ArgumentParser()
-    parser.add_argument("path_to_weights")
-    parser.add_argument("path_to_test")
+app = typer.Typer()
 
-    args = parser.parse_args()
 
-    predict_yolo(args.path_to_weights, args.path_to_test)
+@app.command()
+def test_yolo(
+    weights_path: str = typer.Argument(),
+    test_path: str = typer.Argument(),
```

Comment:
Name of variable `test_path` is unclear. Let's make the name more specific and probably also add a description to these arguments if possible